webpack/scenes/ContentViews/expansions/__tests__/contentViewComponentsModal.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { renderWithRedux, patientlyWaitFor, fireEvent } from 'react-testing-lib-wrapper';
+
+import { nockInstance, assertNockRequest, mockSetting, mockAutocomplete } from '../../../../test-utils/nockWrapper';
+import api from '../../../../services/api';
+
+import RelatedContentViewComponentsModal from '../RelatedContentViewComponentsModal';
+import RelatedCompositeContentViewsModal from '../RelatedCompositeContentViewsModal';
+
+const contentViewComponentsResponse = require('./contentViewComponentsResponse.fixtures.json');
```

Comment:
We are switching to using import instead of require since it helps with some heap size issues in the test suite.
`import contentViewComponentsResponse from './contentViewComponentsResponse.fixtures.json' `